scripts/logrotate.config

```diff
@@ -0,0 +1,11 @@
+/var/log/nginx/*.log {
+        size 400M
+        rotate 10
+        notifempty
+        missingok
+        create 640 nginx nginx
+        compress
+        postrotate
```

Comment:
does this need to be run only once, or once for each log which is rotated?
if it should be only once should it be using the `sharedscripts` directive?